prog/vm/nexus.rb

```diff
@@ -293,6 +297,18 @@ def before_run
     hop_wait_sshable
   end
 
+  def self.setup_vm_options_str(frame)
+    options = []
+    options << "hugepages=off" if frame["hugepages"] == false
+    options << "ch_version=#{frame["ch_version"]}" if frame["ch_version"]
+    options << "firmware_version=#{frame["firmware_version"]}" if frame["firmware_version"]
+    options.join(",")
+  end
```

Comment:
I think per your commit message, it probably does make sense to put this in the entity record. It's not transient  either, it's the kind of thing we would want to report on.